_docs-2/security/on-disk-encryption.md

```diff
@@ -57,6 +91,13 @@ required to perform decryption. The FileDecrypter only has one method:
 ```
 For more help getting started see {% jlink org.apache.accumulo.core.security.crypto.impl.AESCryptoService %}.
 
+## Disabling Crypto
+
+Crypto can be disabled by setting the property `general.custom.crypto.enabled` to false.
```

Comment:
Does this disable for the entire instance? Is there a way to disable for a table when using the PerTableCryptoServiceFactory?

Reply:
Good question, this was something quick that I used just to test if I could disable it last week and have things still work and this is a section that needs to be elaborated on and isn't quite finished. I will take a look tomorrow and see where it gets applied as off the top of my head I don't remember if this only works for instance wide crypto and if there is a different property for per table to disable.

Reply:
See my comment here: https://github.com/apache/accumulo/issues/2930#issuecomment-1249615992